samples/bluetooth/direct_test_mode/src/fem/fem.c

```diff
@@ -291,6 +292,12 @@ int fem_init(NRF_TIMER_Type *timer_instance, uint8_t compare_channel_mask)
 		return -EINVAL;
 	}
 
+#if defined(NRF52840_XXAA) || defined(NRF52833_XXAA)
+	if (mpsl_fem_device_config_254_apply_get()) {
```

Comment:
I am afraid that is not so simple:
* Added an API to set and get a flag that determines whether to apply the DEVICE-CONFIG-254.
  The API user shall set the flag to desired value with :c:func:`mpsl_fem_device_config_254_shall_apply_set`.
  The protocol stacks' responsibility is to check the flag value using :c:func:`mpsl_fem_device_config_254_shall_apply_get` and apply the configuration when the flag value is true

Those two function just set and clear flags. We need our own implementation